packages/components/pages/f-account-info/src/tenants/en-GB.js

```diff
@@ -1,4 +1,32 @@
+const messages = {
+    accountDetails: 'Account Details',
+    yourDetails: 'Your details',
+    contactCustomerCareTeam: "If you'd like to change your email address, get in touch with our customer care team.",
+    deliveryAddress: 'Delivery Address',
+    fields: {
+        emailAddressLabel: 'Email address',
+        emailAddressPlaceholder: 'Your email address...',
+        firstNameLabel: 'First name*',
```

Comment:
In later implementations, we should probably add the asterisks based on whether the property is required or not